.github/workflows/jest.yml

```diff
@@ -16,7 +17,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v1
         with:
-          node-version: "12"
+          node-version: "16"
 
```

Comment:
@AlbertoCasasOrtiz I understand you went back and forth on this, just making sure where did the 16 came from

Reply:
I changed this to 16 since version 12 is deprecated (we have a warning in CI about it), and it is the recommended version by Github now: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/